### DISCUSSION TITLE
Dark mode feature request

### DISCUSSION THREAD

**Discussion by CoderTeam1** [community/community#23972] [Accessibility] [answered]
<p>Please add themes using time like in daytime it is light and in night it is dark</p>

**Comment by CoderTeam1** [comment]
<p>Thanks for the update</p>

**Comment by cehfisher** [comment, marked as answer]
Hello @CoderTeam1 - thank you for your question! Here is a link to a [solution to your issue](https://github.com/orgs/community/discussions/22031#discussioncomment-3235214) posted in another thread. @queenofcorgis - please close this issue as resolved. Thank you!
